Goobi/install/config/messages_de.properties

```diff
@@ -1 +0,0 @@
-info = Goobi Installation der CHANGEME in CHANGEME. <br /> <br /> F\u00FCr Supportanfragen wenden Sie sich bitte an: <a href="mailto:support@intranda.com">support@intranda.com</a>.
```

Comment:
Why was this deleted? It is OK to add new messages, but please don't remove existing ones